### DISCUSSION TITLE
I have Requested for Github student developer pack but it is still pending

### DISCUSSION THREAD

**Discussion by ankitd2000** [community/community#23305] [New to GitHub]
<p>Hello,<br>
I have applied for Github student developer pack on 16th Apr 2020, but my request is still pending.<br>
can you please tell me when my request will be accepted.</p>

**Comment by ghost** [comment, marked as answer]
<p>Hello <a class="mention" href="/ankitd2000">@ankitd2000</a>,</p>
<p>In the interest of privacy and security we aren’t able to discuss account-specific details in a public forum. If you have not yet received an update about your academic discount request we kindly ask that you <a href="https://support.github.com/contact" rel="nofollow noopener">reach out to our Support team for help</a>.</p>
